### PR TITLE
Fix healthcheck issue with Docker containers

### LINKED ISSUE #5010: Docker Healthcheck fails on latest Minio release Docker image
## Expected Behavior
Docker should report the status of Minio container as healthy

## Current Behavior
Docker reports the status as unhealthy

```
CONTAINER ID        IMAGE                                      COMMAND                  CREATED             STATUS                      PORTS               NAMES
e8ce5a03d72a        minio/minio:RELEASE.2017-09-29T19-16-56Z   "/usr/bin/docker-e..."   11 minutes ago      Up 11 minutes (unhealthy)                       zen_turing

```

## Possible Solution
Fix `healthcheck.sh`  

## Steps to Reproduce (for bugs)
1. Launch latest release Docker Minio image
```
sudo docker run -p 9000:9000 --net=host minio/minio:RELEASE.2017-09-29T19-16-56Z server /tmp/test
```
2. After sometime, check the status using `docker ps`. 

## Your Environment

* Version used (`minio version`):
* Environment name and version (e.g. nginx 1.9.1):
* Server type and version:
* Operating System and version (`uname -a`):
* Link to your project:

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
Fix healthcheck issue with Docker containers

## Motivation and Context
Fixes #5010

## How Has This Been Tested?
Manually

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,6 +8,8 @@ ENV CGO_ENABLED 0
 
 WORKDIR /go/src/github.com/minio/
 
+COPY dockerscripts/docker-entrypoint.sh dockerscripts/healthcheck.sh /usr/bin/
+
 RUN  \
      apk add --no-cache ca-certificates && \
      apk add --no-cache --virtual .build-deps git go musl-dev && \
```
Comment (nitisht) [inline Dockerfile]: need to add `curl` here as well, or healthcheck fails

**Comment by poornas** [comment]
I still see the status turn unhealthy after a while
```
➜  ~ docker ps
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS                      PORTS               NAMES
e0c0e5dc8613        minio/minio:test    "/usr/bin/docker-entr"   13 minutes ago      Up 13 minutes (unhealthy)                       big_cori
```

**Comment by harshavardhana** [comment]
> ➜  ~ docker ps
> CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS                      PORTS               NAMES
> e0c0e5dc8613        minio/minio:test    "/usr/bin/docker-entr"   13 minutes ago      Up 13 minutes (unhealthy)                       big_cori

Yeah its because curl is not found.. looks like its being removed in the build.

**Comment by harshavardhana** [comment]
> Yeah its because curl is not found.. looks like its being removed in the build.

Updated should work fine now..

**Review by poornas — APPROVED** [review]
tested - looks good.

**Review by nitisht — APPROVED** [review]
LGTM and Tested

File: dockerscripts/healthcheck.sh [hunk 2]
```diff
@@ -15,6 +15,8 @@
 # limitations under the License.
 #
 
+set -x
```
Comment (lbogdan) [inline dockerscripts/healthcheck.sh:18]: I think this was added for debugging reasons and was forgotten here?

**Pull request merged**